### ISSUE TITLE
InsecureRequestWarning: Unverified HTTPS request is being made

### ISSUE THREAD

**Issue by xeagle2** [SeleniumHQ/selenium#6686]
## 🐛 Bug Report

When I use py binding to open the browser with remote_url='https://' the following warning is always displayed when making a request.

`/usr/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings`

It requires certificates to verify. The recommendation how to handle this is described at https://urllib3.readthedocs.io/en/latest/user-guide.html#ssl

```
http = urllib3.PoolManager(
...     cert_reqs='CERT_REQUIRED',
...     ca_certs=certifi.where())
```

However there is no way to manager this, no way to pass parameters, etc, please have a look at   https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/remote_connection.py#L401

## To Reproduce

The same issue is described at https://stackoverflow.com/questions/51995537/remote-selenium-grid-over-https-is-not-working

## Expected behavior

SSL connection is expected to be be verified without warnings.

## Environment

OS: Linux / Win
Browser: N/A
Browser version: N/A
Browser Driver version: N/A
Language Bindings version: py 3.141.0
Selenium Grid version (if applicable): 3.141.59

**Comment by cgoldberg** [comment]
dupe of #6534

**Comment by vhryhorchuk-amplify** [comment]
@cgoldberg I believe these are different issues:
This bug is about  `InsecureRequestWarning`, and https://github.com/SeleniumHQ/selenium/issues/6534 about `MaxRetryError: HTTPSConnectionPool`
Anyway, it seems neither is fixed in Selenium `3.141.0`
